src/app/shared/models/keyboard-trigger-data.ts

```diff
@@ -0,0 +1,10 @@
+import { Keys } from 'src/app/keyboard/value-objects/key-binding'
+import { KeyEventType } from 'src/app/keyboard/value-objects/key-event-type'
+
+export interface KeyboardTriggerData {
+  keys: Keys
+  actionArguments?: string | number
+  label: string
+  triggerOn: KeyEventType
+  mappedToKeys?: Keys
```

Comment:
We have updated the user facing definition for the mapped keys to be `physical mapping`. Personally I think `physical keys` makes more sense, since the attribute alone is not a mapping. A suggestion is to update this to `physicalKeys` to have it semantically closer to its responsibility. This could be in a different PR.

Reply:
I vote for a different PR ;)